samples/js/browser/avatar/basic.html

```diff
@@ -28,16 +28,6 @@ <h2 style="background-color: white; width: 300px;">Azure Speech Resource</h2>
     <input id="privateEndpoint" type="text" size="64" style="font-size: medium;" placeholder="https://{your custom name}.cognitiveservices.azure.com/"></input><br />
```

Comment:
There is a typo:

for="azureOpenAIEndpoint"

should be:

for="privateEndpoint"

Reply:
same for chat.html